.circleci/cimodel/data/pytorch_build_definitions.py

```diff
@@ -241,13 +241,6 @@ def gen_docs_configs(xenial_parent_config):
             branch="master",
         )
     )
-
-    configs.append(
```

Comment:
Where does this run now? I don't see it in the HUD anywhere (from `test.sh` it looks like the job name needs to have `doc_test`?)

Reply:
should now be `docs_test`, removed in https://github.com/pytorch/pytorch/pull/64201